docs/sources/http_api/alerting.md

```diff
@@ -86,18 +83,47 @@ Authorization: Bearer eyJrIjoiT0tTcG1pUlY2RnVKZTFVaDFsNFZXdE9ZWmNrMkZYbk
 HTTP/1.1 200
 Content-Type: application/json
 {
-  "id": 1,
-  "dashboardId": 1,
-  "panelId": 1,
-  "name": "fire place sensor",
-  "message": "Someone is trying to break in through the fire place",
-  "state": "alerting",
-  "newStateDate": "2016-12-25",
-  "executionError": "",
-  "url": "http://grafana.com/dashboard/db/sensors"
+  "Id": 1,
```

Comment:
This seems like a bug from our side. We should not return json with Capital letters and we should not return all those properties to the user. Please revert this change and I will fix

Reply:
Ok - I hope you aren't going to remove evalData - or maybe we can at least leave it in for the end point here: #11936 

Is the idea that the individual alerting `/api/alerts/:id` should be the same format as a single item from the array returned by `api/alerts/`?